android/KMEA/app/src/main/java/com/keyman/engine/KMManager.java

```diff
@@ -175,6 +201,21 @@ public String toString()  {
 
   private static KMManager.SpacebarText spacebarText = KMManager.SpacebarText.LANGUAGE_KEYBOARD; // must match default given in kmwbase.ts
 
+  // Tiling colored background generated from https://elmah.io/tools/base64-image-encoder/
+  // TODO: Replace with image/css?
```

Comment:
Yes, let's use .png and CSS rather than embedding base64 into the code

Reply:
The reason it's done (base64) on iOS is because the image is dynamically built; there actually isn't a single backing file for the image.  If you're just loading static images in and forwarding those to the banner... yeah, there's little reason to not go the route suggested above.